include/counter.h

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2016 Intel Corporation
+ * Copyright (c) 2018 Nordic Semiconductor
```

Comment:
We use "Nordic Semiconductor ASA" in copyright statements and add new entries at the beginning.

Reply:
fixed